dbt/profiles.yml

```diff
@@ -11,3 +11,22 @@ athena:
       # "database" here corresponds to a Glue data catalog
       database: awsdatacatalog
       threads: 5
+    ci:
+      type: athena
+      s3_staging_dir: s3://ccao-dbt-athena-ci-us-east-1/results/
+      s3_data_dir: s3://ccao-dbt-athena-ci-us-east-1/data/
+      region_name: us-east-1
+      schema: dbt-test
+      database: awsdatacatalog
+      # Prefix all generated data by schema, so that we can delete it when the
+      # PR is merged
+      s3_data_naming: schema_table
```

Comment:
I am actually not totally sure if this works yet, since we're not yet using dbt to manage our CTAs, but the idea here is that eventually when dbt builds CTAs and stores them in S3 the `schema_table` config will tell it to store those files in the configured `s3_data_dir` bucket with the path `{s3_data_dir}/{schema}/{table}/` ([docs](https://dbt-athena.github.io/docs/configuration/table-configuration#table-data-location)). That way when we're ready to have CD clean up the resources generated for the PR, we can use the schema name defined by the `generate_schema_name` to delete everything in `{s3_data_dir}/{schema}/` and leave resources that are being used by other PRs intact.